app/controllers/my-projects/assignment/hearing/add.js

```diff
@@ -86,7 +86,7 @@ export default class MyProjectsProjectHearingAddController extends Controller {
   @action
   async onConfirm() {
     const { dispositions } = this.model;
-    const allActions = this.get('allActions');
+    const allActions = this.get('allActions') || (dispositions.length <= 1);
```

Comment:
Great! I think later on we'll want to centralize the `allActions` state into a computed property. That way we're not re-computing the `allActions` state in different places based on different criteria. Thanks for the quick turnaround!

Reply:
Yes sounds like a good plan